src/Statistics/Acquisition/Aggregator.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace AppBundle\Statistics\Acquisition;
+
+use AppBundle\Statistics\Acquisition\Calculator\CalculatorInterface;
+
+class Aggregator
+{
+    private $calculators = [];
+
+    public function addCalculator(CalculatorInterface $calculator, int $priority = 0): void
+    {
+        $this->calculators[$priority][] = $calculator;
+    }
+
+    public function calculate(StatisticsRequest $request): array
+    {
+        $result = [];
+        foreach ($this->getCalculators() as $calculators) {
+            /** @var CalculatorInterface[] $calculators */
+            foreach ($calculators as $calculator) {
+                $result[] = [$calculator->getLabel() => $calculator->calculate(
+                    $request,
+                    $this->generateDateKeys($request->getStartDate(), $request->getEndDate())
+                )];
+            }
+        }
+
+        return $result;
+    }
+
+    private function generateDateKeys(\DateTimeInterface $startDate, \DateTimeInterface $endDate): array
+    {
+        return array_map(
+            function (\DateTime $date) { return $date->format('Ym'); },
+            iterator_to_array(new \DatePeriod($startDate, new \DateInterval('P1M'), $endDate))
+        );
+    }
+
```

Comment:
done, but with `@return CalculatorInterface[][]` because it's array of calculators with priority:
```
[
  0 => [
    calc1,
    calc2,
  ],
]

```